src/app/modules/forms/ix-forms/components/ix-button-group/ix-button-group.component.ts

```diff
@@ -51,6 +53,10 @@ export class IxButtonGroupComponent implements ControlValueAccessor {
     this.controlDirective.valueAccessor = this;
   }
 
+  @HostBinding('attr.id') get id(): string {
+    return this.formControlName() || '';
```

Comment:
Also you set id here, but also set `ix-label` in `ixRegisteredControl`. Can you use one of these?

Reply:
In some cases, label might not be set. In other cases, id might not be set. So, these give more of a chance that at least one will be set and allow usage for the `ixRegisteredControl` directive.